pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -54,6 +54,29 @@ import (
 )
 
 const (
+	NoConflict = iota
+	// Incoming HTTP existing HTTP
+	HTTPOverHTTP
```

Comment:
The use of "over" is a bit confusing. It reads like it's an application protocol "over" a transport protocol. Maybe we can just separate the protocols in the name with an underscore. (e.g. `Conflict_HTTP_HTTP` or `HTTP_VS_HTTP`)

Reply:
The over is consistent with current conflict metric name. e.g. `ProxyStatusConflictOutboundListenerTCPOverHTTP`